pkg/validations/validations.go

```diff
@@ -173,3 +182,57 @@ func IsValidTag(tag string) bool {
 	tagRegex := `^\w+( \w+)*$` // word characters and whitespace
 	return regexp.MustCompile(tagRegex).MatchString(tag)
 }
+
+// ValidateCaCertificate ensures the specified base64 CA certificate
+// is valid by trying to decode and parse it.
+func ValidateCaCertificate(certificate string) error {
```

Comment:
This should be moved out of the production code and into the test code

Reply:
Yes. IMO it needs to be in a location that is specifically for common test utils.

It doesn't belong in this file as it can be strongly argued that this file is production code.